mkosi

```diff
@@ -1196,7 +1196,7 @@ SigLevel    = Required DatabaseOptional
 
     subprocess.run(cmdline, check=True)
 
-    if not "networkmanager" in args.packages:
+    if not args.packages or not "networkmanager" in args.packages:
```

Comment:
I feel like this would be cleaner the other way around:
```python
if args.packages and "networkmanager" in args.packages:
    enable_networkmanager(workspace)
else:
    enable_networkd(workspace)
```